packages/injector/src/module.ts

```diff
@@ -296,8 +306,11 @@ export class InjectorModule<C extends { [name: string]: any } = any, IMPORT = In
         return this.exports.includes(token);
     }
 
-    isProvided(classType: ClassType): boolean {
-        return isProvided(this.getProviders(), classType);
+    isProvided<T>(token?: ReceiveType<T> | Token): boolean {
```

Comment:
@marcj do you have any clue why the `token` argument being provided here is `[ [Function (anonymous)], 'n!' ]` with the following test?

```ts
test('isProvided supports Type as token', () => {
    interface A {
        b: string;
    }
    const providers = [provide<A>({ useValue: { b: 'a' } })];
    const module = new InjectorModule(providers)
    expect(module.isProvided<A>()).toBe(true);
});
```